### PR TITLE
data structure - first draft

### PR DESCRIPTION
**Pull request by linsydeanna**

### CODE REVIEW

**Comment by vleong2332** [comment]
are these sample data?

**Comment by linsydeanna** [comment]
Yes - just a sample for our reference.

File: data.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const user = {
+  email: 'linsy.joyner@gmail.com',
+  username: 'linsydeanna',
+  password: 'supersecretpassword',
+  _id: '592c8029c1e6f9d97b6b146b',
+  trips: [
+    {
+      name: 'Europe 2017',
```
Comment (vleong2332) [inline data.js:8]: We may need `id` for trips.

File: data.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      departureDate: '10/9/17',
+      returnDate: '10/16/17',
```
Comment (vleong2332) [inline data.js]: Should we call it `startDate` and `endDate`? When I hear `departure` and `return`, I think of flights, which may be true in some cases, but not necessarily accurate for all trips.

File: data.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      cities: [
+        {
+          city: 'Amsterdam',
+          country: 'Netherlands',
+          arrivalDate: '10/10/17',
+          departureDate: '10/13/17',
```
Comment (vleong2332) [inline data.js]: I wonder what your thoughts were on the `arrivalDate` and `departureDate` on cities.

File: data.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+          id: '1234'
+        }
+      ],
+      locations: [
+        {
+          locationName: 'Anne Frank House',
```
Comment (vleong2332) [inline data.js]: Do you think `name` is too vague?

**Review by vleong2332 — COMMENTED** [review]
Is this the "processed" data that will become the state of the app, as opposed to the actual database schema?

File: data.js (hunk 1 above)
Comment (linsydeanna) [reply data.js:8]: Agreed.

File: data.js (hunk 2 above)
Comment (linsydeanna) [reply data.js]: Yes, I agree.

File: data.js (hunk 3 above)
Comment (linsydeanna) [reply data.js]: The dates the user is in that city. Would provide a better breakdown for our agenda view.

File: data.js (hunk 4 above)
Comment (linsydeanna) [reply data.js]: Do you mean `locationName` or `name` of the trip?

File: data.js (hunk 3 above)
Comment (vleong2332) [reply data.js]: Are you thinking these as calculated data or user-inputted data?

File: data.js (hunk 4 above)
Comment (vleong2332) [reply data.js]: `name` instead of `locationName`, since it's already inside the array called `locations`.

Comment (linsydeanna) [reply data.js]: Oh okay. I'll fix that.

File: data.js (hunk 3 above)
Comment (linsydeanna) [reply data.js]: For MVP, everything is user-inputted.

Comment (linsydeanna) [reply data.js]: I will also change these to be `startDate` and `endDate`.

File: data.js (hunk 1 above)
Comment (vleong2332) [reply data.js:8]: I'm glad you agreed with me.

File: src/reducers/user.js [hunk 5]
```diff
@@ -1,13 +1,15 @@
+//rename file
```
Comment (vleong2332) [inline src/reducers/user.js:1]: What are you going to rename it to?

**Comment by linsydeanna** [comment]
Closing and moving data structure to markdown file

**Pull request closed without merging**